packages/terra-alert/src/clinical-lowlight-theme/Alert.module.scss

```diff
@@ -0,0 +1,65 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-alert-color: #b2b5b6;
+    --terra-alert-min-height: 0;
+
+    --terra-alert-icon-font-size: 1.286em;
+    --terra-alert-icon-margin-left: 0;
+    --terra-alert-icon-margin-right: 0.5rem;
+    --terra-alert-icon-margin-top: 0;
+
+    --terra-alert-body-margin-left: 0.64285rem;
+    --terra-alert-body-margin-right: 0.7143rem;
+    --terra-alert-body-margin-top: 0.7143rem;
+
+    --terra-alert-std-margin-bottom: 0.7143rem;
+    --terra-alert-std-margin-right: 0.6667rem;
+
+    --terra-alert-section-padding-bottom: 0.142857rem;
+    --terra-alert-section-padding-top: 0.142857rem;
```

Comment:
Setting this to `0.214285714rem` improves alignment with icons.

Reply:
Updated:
- default-theme - https://github.com/cerner/terra-core/pull/2941/commits/bc9d52db9887bcdb9ce86d86a60fa3c13ce0ce25#diff-3b62f1ed274440a76fc2e27af8d4700bR62
- lowlight-theme - https://github.com/cerner/terra-core/pull/2941/commits/bc9d52db9887bcdb9ce86d86a60fa3c13ce0ce25#diff-15ad366066734dcfa4399a813916cc0cR15